nexus/tests/integration_tests/schema.rs

```diff
@@ -536,7 +536,13 @@ impl InformationSchema {
     fn pretty_assert_eq(&self, other: &Self) {
         // similar_asserts gets us nice diff that only includes the relevant context.
         // the columns diff especially needs this: it can be 20k lines otherwise
+        similar_asserts::assert_eq!(self.tables, other.tables);
```

Comment:
Short answer: yep.

Long answer: The backstory here is that I ran into the anonymous-CHECK-constraints problem a couple of weeks ago (before the change to add the table constraints check landed), and I had trouble diagnosing it because it was impossible to tell which tables were being assigned which opaque IDs. At that point I added my own table constraints check, which helped to make the problem more obvious (the errors had real table names in them!), and then I reordered everything to try to make the most obvious/easiest-to-diagnose failures come first.